api.planx.uk/modules/file/file.test.ts

```diff
@@ -41,39 +42,55 @@ describe("File upload", () => {
   });
 
   describe("Private", () => {
-    const ENDPOINT = "/private-file-upload";
+    const ENDPOINT = "/file/private/upload";
+    const auth = authHeader({ role: "teamEditor" });
+
+    it("returns an error if authorization headers are not set", async () => {
+      await supertest(app)
+        .post("/flows/1/move/new-team")
+        .expect(401)
+        .then((res) => {
+          expect(res.body).toEqual({
+            error: "No authorization token was found",
+          });
+        });
+    });
 
     it("should not upload without filename", async () => {
       await supertest(app)
         .post(ENDPOINT)
+        .set(auth)
         .field("filename", "")
         .attach("file", Buffer.from("some data"), "some_file.txt")
-        .expect(422)
+        .expect(400)
         .then((res) => {
           expect(mockPutObject).not.toHaveBeenCalled();
-          expect(res.body.error).toBe("missing filename");
+          expect(res.body).toHaveProperty("issues");
+          expect(res.body).toHaveProperty("name", "ZodError");
```

Comment:
See above - I'd like there to be more meaningful errors returned here. It does show that the error is the missing `filename` field but it's not in a super friendly format.